packages/upswyng-web/src/components/Hotlines.tsx

```diff
@@ -1,9 +1,101 @@
+import { Container, colors } from "../App.styles";
+import { List, ListItem, ListItemText, Typography } from "@material-ui/core";
+
+import PageBanner from "./PageBanner";
 import React from "react";
 
+const initiatePhoneCall = (phoneNumber: string): void => {
+  window.open(`tel:${phoneNumber}`);
+};
+
+const hotlineList = [
+  {
+    name: "National Suicide Prevention Lifeline",
+    description: "",
+    //"The National Suicide Prevention Lifeline provides free and confidential emotional support to people in suicidal crisis or emotional distress 24 hours a day, 7 days a week, across the United States. The Lifeline is comprised of a national network of over 160 local crisis centers, combining custom local care and resources with national standards and best practices.",
+    phone: "1-800-273-8255",
+  },
+  {
+    name: "National Domestic Violence Hotline",
+    description:
+      "Operating around the clock, seven days a week, confidential and free of cost, the National Domestic Violence Hotline provides lifesaving tools and immediate support to enable victims to find safety and live lives free of abuse. Callers to The Hotline at 1-800-799-SAFE (7233) can expect highly trained, experienced advocates to offer compassionate support, crisis intervention information and referral services in over 170 languages. Our phone and chat services are available to anyone who has been affected by relationship abuse, including those who are currently in abusive relationships, those who are working to heal, friends or family of victims and survivors and anyone in the community who has questions about domestic violence.",
```

Comment:
Ya, these are pretty wordy. What do you think about having some of the descriptions visible and having a show/more less button to reveal the rest?